### ISSUE TITLE
Back in the search - firefox windows

### ISSUE THREAD

**Issue by bastiao** [bioinformatics-ua/catalogue#972]
In firefox at Windows, if you search in the top bar and go back, it retrieves an error.

Reported by Myriam with Firefox at Windows.

Not able to reproduce in Mac OS.

**Comment by ribeiroct** [comment]
In Linux is does not happen either (Firefox 40.0.3)

**Comment by bastiao** [comment]
Are you using "Back" button?

**Comment by ribeiroct** [comment]
Yes, in the browser, I am installing Firefox on a Windows computer to test it out and will get back with a report.

**Comment by ribeiroct** [comment]
Well i tried it on a Windows computer and it is working. 

Perhaps the problem is not going back from the search, but the page we are going back to ? Shloud we inquire further ?

**Comment by bastiao** [comment]
She will be in TC tomorrow. So we can try to ask that. :)
